client/src/components/forms/props.js

```diff
@@ -0,0 +1,39 @@
+const formProps = {
+  style: {
+    textField: {
+      variant: 'outlined',
+      margin: 'dense',
+      fullWidth: true,
+    },
+    textAreaField: {
+      rows: 5,
+    },
+  },
+  html: {
+    addColumn: {
+      title: {
+        id: 'name',
+        label: 'Add Title *',
+        name: 'name',
+        type: 'text',
+      },
+    },
+    editCard: {
+      description: {
+        id: 'desc',
+        name: 'desc',
+        placeholder: 'Type your description here',
+      },
+    },
+    addCard: {
+      title: {
+        id: 'name',
+        label: 'Add Title *',
+        name: 'name',
+        type: 'text',
```

Comment:
For clarification, when you create a **new** card, you can only add a title but not a description as well? Would it be better to be able to add a description as well?

Reply:
If a description is being added during card creation, then nvm about the above; just gonna note that most editing placeholders use the original description as a placeholder

Reply:
You know... I actually find it weird that a card needs a "title".  But to be clear I assume "title" is just the text that shows on the card when looking at the board-ui but when you click on the card to edit it, then you can see a more detailed description in "desc". 

Here's a visual of what that's like in Trello. 
https://imgur.com/a/jMrhJwe

So it doesn't look like you add the description on card creation but only the title.